test/jasmine/tests/colorbar_test.js

```diff
@@ -354,5 +355,49 @@ describe('Test colorbar:', function() {
             .catch(failTest)
             .then(done);
         });
+
+        function getCBNode() {
+            return document.querySelector('.colorbar');
+        }
+
+        it('can drag root-level colorbars in editable mode', function(done) {
+            Plotly.newPlot(gd,
+                [{z: [[1, 2], [3, 4]], type: 'heatmap'}],
+                {width: 400, height: 400},
+                {editable: true}
+            )
+            .then(function() {
+                expect(gd.data[0].colorbar).toBeUndefined();
+                expect(gd._fullData[0].colorbar.x).toBe(1.02);
+                expect(gd._fullData[0].colorbar.y).toBe(0.5);
+                return drag(getCBNode(), -100, 100);
+            })
+            .then(function() {
+                expect(gd.data[0].colorbar.x).toBeWithin(0.591, 0.01);
+                expect(gd.data[0].colorbar.y).toBeWithin(0.045, 0.01);
+            })
+            .catch(failTest)
+            .then(done);
+        });
+
+        it('can drag marker-level colorbars in editable mode', function(done) {
```

Comment:
root-level dragging worked before (the above test), but was not 🔒 by a test. Dragging `marker.colorbar` (this test) was broken.